scripts/prepare_release.sh

```diff
@@ -30,15 +31,17 @@ psed() {
     fi
 }
 
-if [ $# -ne 0 ] && [ "$1" = "--help" ]; then
+if [ $# -eq 0 ] || [ "$1" = "--help" ]; then
```

Comment:
Given that we don't use `-u` mode any more, we shouldn't make it even harder to use the only mode that we use. Please revert "prepare_release.sh: Adjusted logic".

Reply:
I could greatly simplify the script if I remove all arguments, and again I have no problem doing it. But as I recall the whole script was a request from a user who were checking out the tag and wanted to develop on it. The fact that we don't use it is quite out of scope.

Let me remove the logic, but please consider the ask as a requirement, when you are making the python replacement.